### ISSUE TITLE
1.85.6 Builder View Custom Code Name Box Size Too Large

### ISSUE THREAD

**Issue by linussun** [psychopy/psychopy#1673]
When using the Psychopy Builder custom code, the name box is so large that it takes 50% of the vertical space of the window even though there is a single word. This makes the coding windows (begin experiment/routine/each frame/end routine/experiment) too small - and a real pain for the user.

I have downloaded the 1.85.6 builder in Mac OS X 10.11.6 (El Capitan) on iMac (20 inch 2009) 2.66 Intel Core Duo 8GB memory. NVIDIA Geforce 9400 256MB.  This is also the case for 1.85.2 in a Windows 10 environment.

Thanks.

![capture](https://user-images.githubusercontent.com/6187350/35050844-c129a870-fb71-11e7-8d9e-5bc532f483c8.PNG)

**Comment by mmagnuski** [comment]
I have the same problem on Psychopy `1.85.6`, Windows 10.

**Comment by peircej** [comment]
Yes, this is fixed in the repository but an annoyance until the next release. Sorry

**Comment by peircej** [comment]
The fix is in commit fe1cce5a3c

 I think you just need to find the line in psychopy/app/builder/dialogs/dlgsCode.py and change it from
```
sizer1.Add(nameSizer, flag=wx.EXPAND, proportion=1)
```
to 
```
sizer1.Add(nameSizer)
```

**Comment by linussun** [comment]
Perfect, thanks Jon!
